packages/x-components/src/components/sliding-panel.vue

```diff
@@ -72,6 +74,14 @@
     @Prop({ default: true })
     public resetOnContentChange!: boolean;
 
+    /**
+     * CSS classes to add to the buttons.
+     *
+     * @public
+     */
+    @Prop()
+    public buttonClass!: string;
```

Comment:
Shouldn't this be `?` instead of `!` as it it does not have a `default` value and is not `required`, hence optional?